src/FirebaseAuth.js

```diff
@@ -23,9 +23,13 @@
 
   FirebaseAuth = function($firebaseUtils, auth) {
     this._utils = $firebaseUtils;
-    if (typeof ref === 'string') {
-      throw new Error('Please provide a Firebase reference instead of a URL when creating a `$firebaseAuth` object.');
+
+    if (typeof auth === 'string') {
+      throw new Error('The $firebaseAuth service accepts a Firebase auth instance (or nothing) instead of a URL.');
```

Comment:
Here, I feel strongly that we should use the class by name (so it can be looked up in https://firebase.google.com/docs/reference/js).